dut/drivers.wake

```diff
@@ -5,7 +5,7 @@ tuple BasicDriverPlan =
   global VendorName:          String
   global DeviceName:          String
   global LocalMetalDirectory: String
-  global DUTName:             String
+  global IsCompatible:        List (List String) => Boolean
```

Comment:
instead of having every driver write an IsCompatible function, wouldn't it be better to have the driver just supply the list of compatability strings, in order?

Reply:
how do we pick which driver to use if there are multiple matches?

Reply:
if there is an exact match, use that. There should only be one. if not go to the next and match on that. The second string is more generic.

Reply:
so fail if there is more than one match?

Reply:
The spec is vague. 

I would say fail and let it be know why it's failing so that it can be fixed.

Reply:
ok talked with Richard, and your suggestion sounds good. we'll make this a list of strings and just match in order of the compatible list in the dts. if there are multiple matches just pick one and may print a warning.